js/explore.js

```diff
@@ -0,0 +1,19 @@
+//Create the ability to input a word or sentence then output it in reverse. 
+// Do this by reading the word from the last character. Reading the input in reverse, 
+//take each letter and save into a new STRING, forming a new word. 
+//Printing this new word will show the initial word in reverse.
+
+function Reverse(string){
+	char = '';
+	for (var i = string.length; i > 0; i--)
+		char += string[i];
+		return char;
```

Comment:
once a function sees `return` it stops running and returns the request.  This means you're returning the first thing that runs here.